doc/databases.md

```diff
@@ -1,22 +1,129 @@
 # Prepared databases
 
-## GTDB R06-rs202 - DNA databases
+```{contents}
+```
 
-All files below are available under https://osf.io/wxf9z/. The GTDB taxonomy spreadsheet (in a format suitable for `sourmash lca index`) is available [here](https://osf.io/p6z3w/).
+We provide a number of pre-built collections and indexed databases
+that you can use with sourmash.
 
-For each k-mer size, three databases are available.
+## Types of databases
 
-* Zipfile collections can be used for a linear search. The signatures were calculated with a scaled of 1000, which robustly supports searches for ~10kb or larger matches.
-* SBT databases are indexed versions of the Zipfile collections that support faster search. They are also indexed with scaled=1000.
-* LCA databases are indexed versions of the Zipfile collections that also contain taxonomy information and can be used with regular search as well as with [the `lca` subcommands for taxonomic analysis](https://sourmash.readthedocs.io/en/latest/command-line.html#sourmash-lca-subcommands-for-taxonomic-classification). They are indexed with scaled=10,000, which robustly supports searches for 100kb or larger matches.
+For each k-mer size, three types of databases may be available: Zipfile (`.zip`), SBT (`.sbt.zip`), and LCA (`.lca.jzon.gz`).
```

Comment:
```suggestion
For each k-mer size, three types of databases may be available: Zipfile (`.zip`), SBT (`.sbt.zip`), and LCA (`.lca.jzon.gz`).
Unless otherwise specified, all databases are built with the default `scaled` value (DNA scaled=1000).
```